### PR TITLE
Blog page initial improvements

### PR DESCRIPTION
**Pull request by hthi**

### CODE REVIEW

File: src/blog/multiPage.html [hunk 1]
```diff
@@ -59,8 +59,8 @@
               <!-- Logo -->
               <div class="logo-container">
                 <div class="logo-wrap local-scroll">
-                  <a href="#home">
-                    <img class="logo" src="img/logo_light.png" alt="logo">
+                  <a href="/">
+                    <img class="logo logo-block" src="https://s3-us-west-2.amazonaws.com/slack-files2/avatars/2016-02-15/21519932593_9ae1e61096255819a32d_88.png" alt="logo">
```
Comment (unscriptable) [inline src/blog/multiPage.html]: Should we move this image into the codebase?  :)

File: src/blog/multiPage.html [hunk 2]
```diff
@@ -95,7 +95,7 @@
               </div>
             </div> <!-- end col -->
 
-            <div class="menu-socials hidden-sm hidden-xs">
+            <div class="menu-socials hidden-sm hidden-xs" style="display:none;">
```
Comment (unscriptable) [inline src/blog/multiPage.html]: Do you think we should comment these blocks out or remove them from the codebase altogether?

File: src/blog/multiPage.html (hunk 1 above)
Comment (hthi) [reply src/blog/multiPage.html]: Do we want this image here?

File: src/blog/multiPage.html (hunk 2 above)
Comment (hthi) [reply src/blog/multiPage.html]: - If we may implement these blocks of code --> comment out
- If we won't be using them --> remove from codebase

...tbd?

Comment (unscriptable) [reply src/blog/multiPage.html]: I'm a huge fan of deleting, if you're cool with that.  We can always go back to this commit to find the original html.  Thoughts?

**Comment by unscriptable** [comment]
lgtm 👍 

best kind of PR: 
screen shot 2016-12-12 at 11 17 11 am

**Pull request merged**